### PR TITLE
phi1035.phi001

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Updated cts_.xml files
Updated header to current standards.

### CODE REVIEW

**Comment by capitains-hook** [comment]
*[Hook UI build recap](https://ci.perseids.org/repo/PerseusDL/canonical-latinLit/5362)*

File: data/phi1035/phi001/__cts__.xml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" groupUrn="urn:cts:latinLit:phi1035" projid="latinLit:phi001" urn="urn:cts:latinLit:phi1035.phi001" xml:lang="lat">
-            <ti:title xml:lang="eng">Argonautica</ti:title>
+            <ti:title xml:lang="lat">Argonautica</ti:title>
             <ti:edition projid="latinLit:perseus-lat2" urn="urn:cts:latinLit:phi1035.phi001.perseus-lat2" workUrn="urn:cts:latinLit:phi1035.phi001">
-                <ti:label xml:lang="eng">Argonautica</ti:label>
-                <ti:description xml:lang="eng">C. Valerius Flaccus. Otto Kramer. C. Valeri Flacci
-                    Setini Balbi Argonauticon Libri Octo. Leipzig. Teubner. 1913.</ti:description>
+                <ti:label xml:lang="lat">Argonautica</ti:label>
+                <ti:description xml:lang="mul">Valerius Flaccus. Kramer, Otto, editor. C. Valeri Flacci
```
Comment (lcerrato) [inline data/phi1035/phi001/__cts__.xml:5]: We usually do [primary author.] [frontpiece book title.] [editor/translator.] [pub place: publisher, year.]
It looks like the editor is out of order here.

**Review by lcerrato — CHANGES_REQUESTED** [review]

File: data/phi1035/phi001/__cts__.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/phi1035/phi001/__cts__.xml:5]: I will fix that.

**Comment by capitains-hook** [comment]
*[Hook UI build recap](https://ci.perseids.org/repo/PerseusDL/canonical-latinLit/5373)*

**Review by lcerrato — APPROVED** [review]

**Pull request merged**